CMakeLists.txt

```diff
@@ -239,6 +241,14 @@ else()
   find_package( hipblas-common REQUIRED CONFIG PATHS ${HIP_DIR} ${ROCM_PATH} /opt/rocm)
 endif()
 
+if(HIPBLASLT_ENABLE_MARKER)
+  find_library(rocTracer roctx64)
+  if(NOT rocTracer)
+    message(FATAL_ERROR "roctracer not found, but HIPBLASLT_ENABLE_MARKER is enabled")
+  endif()
+  add_definitions(-DHIPBLASLT_ENABLE_MARKER)
```

Comment:
miss this part https://github.com/ROCm/MIOpen/blob/61d9825a5eaec632a727e30d4ea881f28c38626f/CMakeLists.txt#L533

Reply:
why we need this

Reply:
I don't think we need this since we only use roctx

Reply:
Done by adding rocm_package_add_dependencies instead